src/store_client.cc

```diff
@@ -315,14 +341,8 @@ storeClientCopy2(StoreEntry * e, store_client * sc)
      * dropped events
      */
 
-    if (sc->flags.copy_event_pending) {
```

Comment:
The comment above talks about "dropped events". It may seem like that comment should be removed now because we are no longer using event.h events anymore. However, I think the comment is still valid. AFAICT, here, the word "event" does not mean an eventAdd() event but any _notification_ about response data availability that may reach this code. We may still "drop" (i.e. not react to) those notifications due to the flags.store_copying check.

Roughly speaking, the official code "dropped" the second such conflicting notification[^1] (while queuing the first via eventAdd()). Our code drops the very first such conflicting notification[^1]. Since both the official code and PR code should answer each storeClientCopy() request with the first-available information, queuing additional notifications seems pointless and dangerous.

[^1]: ... and all the subsequent ones for the current flags.store_copying (i.e. doCopy() is in progress) state.

Reply:
> AFAICT, here, the word "event" does not mean an eventAdd() event but any _notification_ about response data availability that may reach this code. We may still "drop" (i.e. not react to) those notifications due to the flags.store_copying check.
> 

Assuming you are right, I suggest updating the comment to reflect current reality with  s/event/notification/ instead of just leaving it.